src/utils.js

```diff
@@ -0,0 +1,18 @@
+exports.deleteMessages = function (channel, deleteAmount) {
```

Comment:
An arrow function again here?

Reply:
Arrow function only changes the "this" context, but I'll change it to an arrow function for readability.